init.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
```

Comment:
```suggestion
#!/bin/bash
```
If we don't update this, executing the script directly via `./init.sh` might fail depending on the shell.